package/MDAnalysis/analysis/align.py

```diff
@@ -246,9 +246,13 @@ def rotation_matrix(a, b, weights=None):
     :meth:`MDAnalysis.core.groups.AtomGroup.rotate` to generate a rotated
     selection, e.g. ::
 
-    >>> R = rotation_matrix(A.select_atoms('backbone').positions,
-    >>>                     B.select_atoms('backbone').positions)[0]
-    >>> A.atoms.rotate(R)
+    >>> from MDAnalysisTests.datafiles import TPR, TRR
+    >>> from MDAnalysis.analysis import align
+    >>> A = mda.Universe(TPR, TRR)
+    >>> B = A.copy()
+    >>> R = align.rotation_matrix(A.select_atoms('backbone').positions,
+    ...                           B.select_atoms('backbone').positions)[0]
+    >>> A.atoms.rotate(R) # doctest: +SKIP
```

Comment:
Why do we have to skip the doc test here? We'd like to check that _everything_ works.

Doc test expects an output here so just add it: figure out what `A.atoms.rotate(R)` prints and then add it as a line after (and remove the +SKIP).